src/components/dropdown-filter-ajax/dropdown-filter-ajax.js

```diff
@@ -158,9 +158,11 @@ class DropdownFilterAjax extends DropdownFilter {
    */
   handleBlur = () => {
     if (!this.blockBlur) {
-      const filter = this.props.create ? this.state.filter : null;
       // close list and reset filter
-      this.setState({ open: false, filter });
+      this.setState(prevState => ({
+        open: false,
+        filter: this.props.create ? prevState.filter : null
```

Comment:
Is this just because it was a ternary operator?

Reply:
It is bad to `setState` with existing state as it is not always a synchronous event, so we needed to use the function version.